ompi/mca/pml/ob1/pml_ob1_hdr.h

```diff
@@ -84,15 +86,9 @@ struct mca_pml_ob1_match_hdr_t {
     int32_t  hdr_src;                      /**< source rank */
     int32_t  hdr_tag;                      /**< user tag */
     uint16_t hdr_seq;                      /**< message sequence number */
-#if OPAL_ENABLE_HETEROGENEOUS_SUPPORT
     uint8_t  hdr_padding[2];               /**< explicitly pad to 16 bytes.  Compilers seem to already prefer to do this, but make it explicit just in case */
-#endif
 };
-#if OPAL_ENABLE_HETEROGENEOUS_SUPPORT
 #define OMPI_PML_OB1_MATCH_HDR_LEN  16
```

Comment:
Do we still need this or can we now use sizeof(mca_pml_ob1_match_hdr_t) ?

Reply:
By my count it's 16, so I don't think so.